test/integration/util/kubectl_runner.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	commonutil "k8s.io/minikube/pkg/util"
+)
+
+// KubectlRunner runs a command using kubectl
+type KubectlRunner struct {
+	T          *testing.T
+	BinaryPath string
+	KCtx       string
+}
+
+// NewKubectlRunner creates a new KubectlRunner
+func NewKubectlRunner(t *testing.T, kctx string) *KubectlRunner {
+	p, err := exec.LookPath(kubectlBinary)
+	if err != nil {
+		t.Fatalf("Couldn't find kubectl on path.")
+	}
+	return &KubectlRunner{BinaryPath: p, T: t, KCtx: kctx}
+}
+
+// CurrentContext gets the current kubectl current-context
```

Comment:
Since `PastContext` makes no sense, can this be named `Context` or `ContextName`. I do understand that this differs from the name of the kubectl command.